databases/databases-for-redis/concepts.mdx

```diff
@@ -0,0 +1,115 @@
+---
+meta:
+  title: Databases for Redis™  - Concepts
+  description: This page explains all the concepts related to Databases for Redis™ .
+content:
+  h1: Concepts
+  paragraph: This page explains all the concepts related to Databases for Redis™ .
+tags: advanced settings
+categories:
+  - database
+---
+
+<Concept opened>
+
+## Acknowledgement flag (ACKs)  
+
+Are [TCP](https://en.wikipedia.org/wiki/Transmission_Control_Protocol) flags used to indicate that a packet was received successfully.
+
+</Concept>
+
+<Concept opened>
+
+## Allowed IPs
+
+Access control list (ACL) rules [define permissions for remote access to a Database Instance](/databases/databases-for-postgresql-and-mysql/how-to/manage-allowed-ip-addresses/). A rule consists of an IP address or an IP range.
+
+</Concept>
+
+<Concept opened>
+
+## Cluster 
+
+A Redis™ cluster consists of a set of primary and secondary nodes. The cluster nodes use hash partitioning to split the keyspace into 16,384 key slots. Each master of the cluster is responsible for a subset of those slots. Each secondary node replicates the data of one of the primary ones.
+
+</Concept>
+
+<Concept opened>
+
+## Cluster mode
+
+Each Managed Database for Redis™ cluster consists of three compute Instances hosting each a primary node and a secondary node for one of the other nodes. If one of the primary nodes fails, the remaining nodes hold a vote and elect one of the remaining secondary nodes as the new primary node. When the failing node rejoins the cluster, it automatically becomes a secondary node and begins to replicate the data of another primary node. 
+
+</Concept>
+
+<Concept>
+
+## Endpoint
+
+A point of connection to a database. The endpoint is associated with an IPv4 address and a port, and contains the information of whether the endpoint is read-write or not.
+
+</Concept>
+
+<Concept>
+
+## High availability
+
+Is an option that allows you to create a secondary server, with an up-to-date replica of the database. If the primary server fails for any reason, the secondary can take over requests, reducing downtime.
```

Comment:
```suggestion
The High Availability is handled through Redis™ clustering feature. This is a 3 nodes multi master architecture. In case of downtime on 1 node of the cluster, a new node is able to take over the requests with a minimal downtime.
```